src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/AddOrRemoveDirectorsControllerTest.java

```diff
@@ -27,16 +27,14 @@
 import org.springframework.mock.web.MockHttpSession;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.setup.MockMvcBuilders;
-import org.springframework.ui.Model;
-import org.springframework.validation.BindingResult;
 import org.springframework.web.servlet.view.UrlBasedViewResolver;
 import uk.gov.companieshouse.web.accounts.exception.ServiceException;
```

Comment:
This there a way to increase coverage for submitAddOrRemoveDirectorsController. When you run unit test with coverage in intelliJ it does not look great for the method submitAddOrRemoveDirectors ?

Reply:
There is a way, but i struggled with it. If you (or someone next week) can help me with it it'll only take one more test to increase coverage

Reply:
Done, controller at 100% coverage 👍